includes/html/api_functions.inc.php

```diff
@@ -1074,9 +1074,13 @@ function search_ports(Illuminate\Http\Request $request)
 {
     $field = $request->route('field');
     $search = $request->route('search');
+    $columns = $request->get('columns');
+    if ($validate = validate_column_list($columns, 'ports') !== true) {
```

Comment:
Might need to be this:
`if (($validate = validate_column_list($columns, 'ports')) !== true) {`

Otherwise, maybe you are looking for this? Your current code will always return true if it enters the if statement.
```php
if (! validate_column_list($columns, 'ports')) {
    return false;
}
```